src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(getName() + "The Bulldozer began its work");

```

Comment:
```suggestion
        System.out.println(getName() + " The Bulldozer began its work");

```

Иначе имя сольется с остальным текстом